app/views/registrations/_form.html.haml

```diff
@@ -2,6 +2,7 @@
 = simple_form_for([@event, @registration], validate: true) do |f|
   = f.input :started_at, as: :hidden, input_html: { value: @registration.started_at }
   = f.input :step_to_validate, as: :hidden, input_html: { value: "start" }
+  = f.input, :user_id, as: :hidden, input_html: { value: current_user.try(:id) }
```

Comment:
@willblake01 -- this is importat. I suggest doing some reading on how `try` works

Reply:
Will do!